modules/network-observability-operator-install.adoc

```diff
@@ -5,7 +5,11 @@
 :_content-type: PROCEDURE
 [id="network-observability-operator-installation_{context}"]
 = Installing the Network Observability Operator
-You can install the Network Observability Operator using the {product-title} web console Operator Hub. When you install the Operator, you also create the `FlowCollector` resource. You can set specifications in the web console when you create the  `FlowCollector`.  
+You can install the Network Observability Operator using the {product-title} web console Operator Hub. When you install the Operator,  it provides the `FlowCollector` custom resource definition (CRD). You can set specifications in the web console when you create the  `FlowCollector`.  
+
+.Prerequisites
+
+* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], in which you can create a LokiStack instance using `openshift-network` configuration mode. It provides fully automatic, in-cluster authentication and authorization support for the Network Observability Operator. 
```

Comment:
Same suggestion as the other paragraph 🙂

Reply:
Ok - rather than repeating this info, I'm going to remove the "in which you can create a LokiStack...." in these prerequisites and leave it in the Loki Installation description.